app/models/manageiq/providers/ibm_cloud/container_manager/container.rb

```diff
@@ -0,0 +1,4 @@
+ManageIQ::Providers::Kubernetes::ContainerManager.include(ActsAsStiLeafClass)
```

Comment:
Ah sorry I didn't fully explain this.

So if we subclass a model (aka it has a backing database table, not just a ruby class) from another plugin, we have to declare that we are doing this so that STI queries don't pull back both (in this case) `Kubernetes::ContainerManager::Container` _and_ `IbmCloud::ContainerManager::Container` types.

We do this by including the `ActsAsStiLeafClass` into the class we're inheriting from.

Long story short, this should be `ManageIQ::Providers::Kubernetes::ContainerManager::Container.include(ActsAsStiLeafClass)`

Reply:
Makes sense, thanks for the explanation